src/routes/index.ts

```diff
@@ -63,12 +64,14 @@ export const setup = (app: Application) => {
 		'/device/v2/:uuid/state',
 		gracefullyDenyDeletedDevices,
 		apiKeyMiddleware,
+		registerDeviceContact('params.uuid'),
```

Comment:
This allows the param to be chosen from the request, but this shouldn't really change. I would be happy to make this statically coded to the `:uuid` param if others are in agreement.